plugins/defaultplugins/aclplugin/data_resync.go

```diff
@@ -17,11 +17,21 @@ package aclplugin
 import (
 	"github.com/ligato/vpp-agent/plugins/defaultplugins/aclplugin/model/acl"
 	"github.com/ligato/cn-infra/logging"
+	"time"
 )
 
 // Resync writes ACLs to the empty VPP
 func (plugin *ACLConfigurator) Resync(acls []*acl.AccessLists_Acl, log logging.Logger) error {
 	log.Debug("Resync ACLs started")
+	// Start time measuring
+	start := time.Now()
+	// Calculate and log bfd resync
+	defer func() {
+		if plugin.stopwatch != nil {
+			plugin.stopwatch.Overall = time.Since(start)
```

Comment:
hmmm, strange to have two lines that are really tight together in a way:
- first line sets some field
- second line uses behind the scenes the field behinde the scences in a function call